xclim/indices/_threshold.py

```diff
@@ -68,9 +69,47 @@
     "maximum_consecutive_wet_days",
     "sea_ice_area",
     "sea_ice_extent",
+    "windy_days",
 ]
 
 
+@declare_units(pr="[speed]", thresh="[speed]")
+def calm_days(
+    pr: xarray.DataArray, thresh: str = "2 m/s", freq: str = "MS"
+) -> xarray.DataArray:
+    r"""Calm days.
+
+    The number of days with average windspeed below threshold.
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
```

Comment:
I would call this signature `wnd` or `sfcWnd` to not confuse it with precipitation

Reply:
But shouldn't these methods have the same signature style as others like "dry_days" or "wet_days"?

Reply:
We try to make it so that the signature style (IE: the variable being passed) makes reference to the CMIP variable name that it's examining (for examples: https://pcmdi.llnl.gov/mips/cmip3/variableList.html). Generic indices that work on run-lengths of any value tend to use `da`, temperature-based indicators use `tas`, precipitation uses `pr`, etc.

We have some wind-based indices that consider eastward and northward wind (`uas` and `vas`). Since this is examining wind-speed (not direction), I'd suggest using `sfcWind` (CF-1.0 name: "Near-Surface Wind Speed"; units: "m s-1"; standard name: "wind_speed").

Reply:
Ah, I thought you were referring to the method name part of the signature rather than the parameter name. Sure, `sfcWind` sounds good. I'll update that.

Reply:
My mistake! Thanks!